code/modules/mob/living/simple_mob/subtypes/animal/giant_spider/webslinger.dm

```diff
@@ -22,8 +22,8 @@
 	icon_state = "webslinger"
 	icon_living = "webslinger"
 	icon_dead = "webslinger_dead"
-	maxHealth = 90
-	health = 90
+	maxHealth = 60 //VOREStation edit
+	health = 60 //VOREStation edit
```

Comment:
So, it just occurred to me, why are all of these marked as vorestation edits when you could have just added a new file and plonked them all in?

Reply:
We already have a zz_vore_overrides.dm where these can be tacked into